lib/gitlib.sh

```diff
@@ -103,17 +129,24 @@ gitlib::last_releases () {
   declare -Ag LAST_RELEASE
 
   logecho -n "Setting last releases by branch: "
-  for release in $($GHCURL $K8S_GITHUB_API/releases|\
+  for release in $($GHCURL $K8S_GITHUB_API_ROOT/$PROJECT/$PROJECT/releases |\
```

Comment:
I don't understand this change, and I think it's wrong.

Reply:
Left a comment in the code.  Releases are always defined by the top level org (for now).